src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -219,7 +219,7 @@ private function replaceClassKeyword(Tokens $tokens, $classIndex)
 
         $tokens->insertAt($classBeginIndex, new Token(array(
             T_CONSTANT_ENCAPSED_STRING,
-            "'".$this->makeClassFQN($classImport, $classString)."'",
+            '\''.$this->makeClassFQN($classImport, $classString).'\'',
```

Comment:
:-1: for this changes

Reply:
You mean only `"'"` or the entire PR?

Reply:
we follow Sf rule here:
use single-quoted string except of cases when using it would require extra escaping

Reply:
If so, I'm going to make this configurable and disabled by default.

Reply:
for which fixer?

Reply:
`SingleQuoteFixer`: I'll rebase this PR on master instead of 2.2, and create a configuration whether to fix or not double-quoted strings that contain single-quotes; disabled by default for Symfony and BC compatibility